libs/klv/src/main/java/org/codice/alliance/libs/klv/AttributeNameConstants.java

```diff
@@ -22,17 +28,19 @@
  */
 public class AttributeNameConstants {
 
-    public static final String TEMPORAL_START = "temporal.start";
+    public static final String TEMPORAL_START = DateTime.START;
```

Comment:
It seems odd to me to redeclare these constants.  Why not just use them directly?